.github/workflows/continuous-integration.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         with:
           node-version: 12
       - name: Cache dependencies
-        uses: actions/cache@v1
+        uses: actions/cache@v2
         with:
           key: npm-${{ hashFiles( 'package-lock.json') }}
```

Comment:
Nope

Reply:
fixed in https://github.com/ridedott/back-end-assignment-boilerplate/pull/192/commits/d2e9be3d7774ca5e6e8caf843842a0961aaa0c82